source/includes/administration/_organizations.md

```diff
@@ -818,4 +818,124 @@ Returns an HTTP status code 200, with an empty response body.
 ```shell
 # Retrieve the organization's billing information
 curl -X DELETE "https://cloudmc_endpoint/api/v1/organizations/87895f43-51c1-43cc-b987-7e301bf5b86a/billing_info" \
-   -H "MC-Api-Key: your_api_key"
\ No newline at end of file
+   -H "MC-Api-Key: your_api_key"
+```
+
+
+<!-------------------- UPDATE BILLABLE ORGANIZATION INFORMATION -------------------->
+### Set billable organization information
+`PUT /organizations/:organization_id/billable`
+
+Set the organization to billable and update the billable organization information.
+
+```shell
+# Update billable organization info
+curl -X PUT "https://cloudmc_endpoint/api/v1/organizations/87895f43-51c1-43cc-b987-7e301bf5b86a/billable" \
+   -H "MC-Api-Key: your_api_key" \
+   -H "Content-Type: application/json" \
+   -d "request_body"
+```
+> Request body example:
+
+```json
+{
+   "id": "39f6260d-f0ca-4be9-93c4-46405d471c04",
+   "billableStart": "2015-01-18",
+	"pricingPackage": {
+		"id": "58bad1de-1f87-422b-b44f-27ed58889272"
+	}
+}
+```
+
+Required | &nbsp;
+---- | ----
+`id`<br/>*UUID* | The id of the udpated billable organization information.
+`billableStart`<br/>*string*  | The date the organization became billable, in YYYY-MM-DD format.
+`pricingPackage.id`<br/>*UUID* | The id of the pricing package applied on the organization.
+
+<!-------------------- LIST APPLICABLE PRICING PACKAGES TO ORG -------------------->
+### List applicable pricing packages to an organization
+
+`GET /organizations/:organization_id/pricing_packages`
+
+Retrieves a list of applicable pricing packages to an organization.
+
+```shell
+# Retrieve visible organizations
+curl "https://cloudmc_endpoint/api/v1/organizations/87895f43-51c1-43cc-b987-7e301bf5b86a/pricing_packages" \
+   -H "MC-Api-Key: your_api_key"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "offerType": "30-day-trial",
+      "pricingDefinition": {
+        "id": "c9a396f1-690a-4201-a3bf-b1da37221acc"
+      },
+      "scopeOrganization": {
```

Comment:
seems like the applied pricing information and not the pricingPackage. There should be no scopeOrganization, no?

Reply:
0a2d885bbed51edfc5051afc4ce786fa623a3d7f